firstvoices/backend/views/contact_us_views.py

```diff
@@ -0,0 +1,174 @@
+            validate_email(from_email)
+
+            # If the site does not have any contact emails or users, fallback to the default emails in AppJson
+            if to_email_list is None or len(to_email_list) == 0:
+                logger.warning(
+                    f'No emails found in the contact_email and contact_users fields found on site "{site.title}".'
+                )
+                to_email_list = get_fallback_emails()
+
+            # If the site has excluded words, validate that the message does not contain any of them
+            self.validate_no_excluded_words(message, name, from_email, site)
+
+            try:
+                # Format the final email
+                final_message = (
+                    "The following message was sent from the contact us form on the FirstVoices website:\n\n"
+                    f"Name: {name}\n"
+                    f"Email: {from_email}\n\n"
+                    f"Message:\n{message}\n"
+                )
+                send_mail(
+                    subject=f"Contact Us Form Submission from {name} ({from_email})",
+                    message=final_message,
+                    recipient_list=to_email_list,
+                    from_email=settings.DEFAULT_FROM_EMAIL,
+                )
+            except (ConnectionRefusedError, SMTPException) as e:
+                logger.error(f"Contact us email failed to send. Error: {e}")
+                raise ContactUsError()
+
+            return Response(
+                {"message": "The email has been successfully sent."}, status=202
+            )
+        except Exception as e:
+            raise ContactUsError(e.message) if hasattr(
+                e, "message"
+            ) else ContactUsError()
+
+    @staticmethod
+    def validate_no_excluded_words(message, name, from_email, site):
+        """
+        Validates that the email, name, and message do not contain any excluded words. If the message contains any
+        excluded words, an error is raised. The excluded words list can be set in the AppJson model with the key
+        'contact_us_excluded_words'.
+        """
+
+        logger = logging.getLogger(__name__)
+        excluded_words = AppJson.objects.filter(key="contact_us_excluded_words")
+        if excluded_words.count() > 0:
+            word_list = excluded_words.first().json
+
+            # Validate that the excluded words are a list of strings
+            if not isinstance(word_list, list) or not all(
+                isinstance(word, str) for word in word_list
+            ):
+                logger.error(
+                    'The "contact_us_excluded_words" field in AppJson model must be a list of strings.'
+                )
+                raise ContactUsError()
+
+            # Validate that the message does not contain any excluded words
+            for word in word_list:
+                if re.search(word.lower(), (message + name + from_email).lower()):
```

Comment:
Let's try this out... we may need to add some word boundaries to the search criteria so we don't match on the middle of words etc. E.g., if we banned the word "laugh" and someone used "manslaughter", or their email was based on a name like Lau Ghislaine or whatever.